waku/v2/protocol/waku_rln_relay/waku_rln_relay_types.nim

```diff
@@ -42,6 +42,8 @@ type WakuRLNRelay* = object
 # TODO may be able to make these constants private and put them inside the waku_rln_relay_utils
 const 
   MembershipFee* = 5.u256
-  Depth* = 32.u256
+  #  the current implementation of the rln lib only supports a circuit for Merkle tree with depth 32
+  MerkleTreeDepth* = 32
```

Comment:
I think we can probably set this to 20

Also I saw 64 before but not sure what that is referring to?

Reply:
As for the tree height, I am still communicating with the rln team, once I figure out whether the size is fixed or not, I will update this part as well.

64 was the size of the Poseidon hash output in hex, in my new commits I replaced it with a proper const variable.